### ISSUE TITLE
Should support multi language for git_sync

### ISSUE THREAD

**Issue by datphan** [teracyhq/dev#580]
Example in VN, the `git status` will display:
```
Trên nhánh master
Nhánh của bạn đứng trước "origin/master" 18 lần chuyển giao.
  (dùng "git push" để xuất bản các lần chuyển giao nội bộ của bạn)
```

and git check will not works

**Comment by datphan** [comment]
maybe add a prefix: `LANG=en_GB`:
```
LANG=en_GB git status
```

**Comment by hoatle** [comment]
need @hoavt to confirm if the change works.

**Comment by hoatle** [comment]
we'll port this fix for v0.6.0-a5 to release v0.6.0-a6

**Comment by hoatle** [comment]
ported: https://github.com/teracyhq/dev/commit/625defab85059c5f8a6bc1741797a7de18cbf214
